package.json

```diff
@@ -54,8 +54,8 @@
         "size-limit": "^6.0.3",
         "tslint": "^6.1.3",
         "typescript": "^4.5.4",
-        "webpack": "4.35.0",
-        "webpack-cli": "3.3.5",
+        "webpack": "^5.72.0",
+        "webpack-cli": "^4.9.2",
         "webpack-dev-server": "3.7.2"
```

Comment:
Need to add size limit check on these bundles (in know that there is already a size limit check from the recipe folders, but is always going to be equal to the bundle's size? If yes, then ignore this comment)

Reply:
Its always super close to the recipe folder sizes (If email verification is 2kb, the bundle is ~1.99 - 2.01 kb). I dont think the size limts for bundles are required

https://github.com/supertokens/supertokens-web-js/runs/6034725665?check_suite_focus=true